docs/create-your-own-dapp.md

```diff
@@ -24,15 +24,20 @@ $ yarn install && yarn dist
 ```
 
 Creating a dApp can be done by building a simple single-page web application.  To get started
-quickly, clone or fork the [nOS dApp Starter Kit](https://github.com/nos/dapp-starter-kit), and
-follow the documentation there.
+quickly, install the [@nosplatform/create-nos-dapp package](https://www.npmjs.com/package/@nosplatform/create-nos-dapp) and follow the documentation there:
+```
+npm install -g @nosplatform/create-nos-dapp
+```
+and generate a new project using the `create-nos-dapp` command.
+
+clone or fork the [nOS dApp Starter Kit](https://github.com/nos/dapp-starter-kit), 
```

Comment:
This can be left out as it's basically the job of the CLI tool xD

Reply:
Yeah wuh I thought I removed this.